frontend/public/components/graphs/tooltip.tsx

```diff
@@ -43,7 +43,7 @@ export const ChartLegendTooltipContent: React.FunctionComponent<ChartLegendToolt
   theme = getTheme(themeColor, themeVariant),
   ...rest
 }) => {
-  const pointerLength = theme && theme.tooltip ? theme.tooltip.pointerLength : 10;
+  const pointerLength = theme && theme.tooltip ? (theme.tooltip.pointerLength as number) : 10;
```

Comment:
Why is `as number` needed now? Have we declared the types incorrectly somewhere.

Reply:
The tooltip is now defined as: 
```
tooltip?: {
    style?: VictoryLabelStyleObject | VictoryLabelStyleObject[];
    flyoutStyle?: VictoryStyleObject;
    cornerRadius?: NumberOrCallback;
    pointerLength?: NumberOrCallback;
    flyoutPadding?: PaddingOrCallback;
    flyoutWidth?: NumberOrCallback;
    flyoutHeight?: NumberOrCallback;
    orientation?: OrientationOrCallback;
    pointerOrientation?: OrientationOrCallback;
  };
```
before it was defined as number

Reply:
We should update the console type in that case so it matches.

Reply:
@jcaianirh FYI [this PR](https://github.com/patternfly/patternfly-react/pull/5245) was just merged in PatternFly to resolve errors @dtaylor113 found in testing the PF prerelease related to `tooltip.pointerLength` - not quite sure what it may mean for you here but just wanted to bring it to your attention.

Reply:
Just pushed some [code](https://github.com/openshift/console/compare/master...dtaylor113:pf-prerelease-v4.70.2#diff-bcdbb5510e4a564be7d670a8518cd7574490eeb4c0473b40ae2cc8e096ea5625R47) to address this with latest PF prerelease.

Reply:
Updated with @dtaylor113 's suggestion using Helpers.evaluateProp